### ISSUE TITLE
Upgrading

### ISSUE THREAD

**Issue by Chappie88** [elewin/pi-weather-station#14]
Sorry, a rookie when it comes to RPI commands. Is there a way to upgrade to v2 without starting over? I have tried the sudo apt-get upgrade with no luck.

**Comment by chrism583** [comment]
I came from a similar place as you with RPI commands but I was able to successfully update following these steps:
SSH into your pi, cd into your pi-weather-station folder, and run "sudo git pull origin master".  Initially I got commit errors but was able to resolve those by running "sudo git reset --hard" and then re-running "sudo git pull origin master".  Remember to register for a ClimaCell v4 API because you'll need to enter that new key after the update.

**Comment by elewin** [comment]
You're going to have to pull down the latest version of the repo.

**Comment by Chappie88** [comment]
Have you noticed any advantages with the newer software setup that makes the upgrade worth it?

    On Friday, April 9, 2021, 4:45:48 PM CDT, chrism583 ***@***.***> wrote:  
 
 


I came from a similar place as you with RPI commands but I was able to successfully update following these steps:
SSH into your pi, cd into your pi-weather-station folder, and run "sudo git pull origin master". Initially I got commit errors but was able to resolve those by running "sudo git reset --hard" and then re-running "sudo git pull origin master". Remember to register for a ClimaCell v4 API because you'll need to enter that new key after the update.
